src/main/java/seedu/address/model/consultation/Time.java

```diff
@@ -55,6 +55,20 @@ public static boolean isValidTime(String test) {
         }
     }
 
+    /**
+     * Compares this time to another time based on the time of day.
+     *
+     * @param otherTime The other time to compare to.
+     * @return A negative integer, zero, or a positive integer if this time is before, equal to,
+     *         or after the specified time.
+     */
+    public int compareTo(Time otherTime) {
```

Comment:
Since this is already here, might as well make the class implement `Comparable<Time>`. I think it allows more things to be done, such as using Java's built-in sorting for arrays etc.